src/translations/locale/locale.en-US.json

```diff
@@ -1,4 +1,27 @@
 {
+  "App.about.us": "About Us",
+  "App.company": "Company",
+  "App.imprint": "Imprint",
+  "App.privacy.policy": "Privacy policy",
+  "App.signup": "Sign up",
+  "App.login": "Login",
+  "App.email": "E-Mail",
+  "App.password": "Passwort",
+  "App.link.about.us": "https://www.everreal.co/en-US/about-us",
+  "App.link.imprint": "https://www.everreal.co/en-US/imprint",
+  "App.link.privacy.policy": "https://www.everreal.co/en-US/privacy-policy",
```

Comment:
Where would you see that `baseUrl` should be coming from? Keep in mind that this will be an authentication/authorization micro-service that will be hosted on a subdomain (like auth.everreal.co), and these links should point out to the current website that will host these pages. Do you mean that `baseUrl` should be added in configuration?